meme_bot_client.py

```diff
@@ -9,6 +9,10 @@ async def on_message(self, message):
         # don't respond to ourselves
         if message.author == self.user:
             return
-        
-        await MemePlayer.play_meme(message)
+        try:
+            await MemePlayer.play_meme(message)
+        except discord.errors.ClientException:
+            await message.channel.send(file=discord.File('pics/slowDownNeighbor.jpg'))
+        except Exception as e:
+            print(e)
```

Comment:
So I think we can do a few things better here:
1. ClientException could happen for a variety of reasons, not just bc voice is already in use. I think we should use `slowDownNeighbor` only for specific messages.

2. I think we can handle the slowdown neighbor exception on the exact line of code it happens at. Reduce of scope of our trycatch.

3. On generic errors, we should try to give a message to the user